pkg/sfs/controllerserver.go

```diff
@@ -40,46 +45,64 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 
 	client, err := cs.Driver.cloud.SFSV2Client()
-    if err != nil {
+	if err != nil {
 		klog.V(3).Infof("Failed to create SFS v2 client: %v", err)
 		return nil, status.Error(codes.Internal, err.Error())
-    }
+	}
+
+	// check pv hasCreate or not
+	if value, ok := pvcProcessSuccess[req.Name]; ok && value != nil {
```

Comment:
why use `pvcProcessSuccess` ? The data in memory is unreliable, it will lose after restarting